apl-core/src/test/resources/db/data.sql

```diff
@@ -74,10 +74,10 @@ INSERT into Public.TRANSACTION_SHARD_INDEX(transaction_id, partial_transaction_h
 (102,X'b96d5e9f64e51c597513717691eeeeaf18a26a864034f62c' ,1),
 (103,X'cca5a1f825f9b918be00f35406f70b108b6656b299755558' ,1)
 ;
-INSERT into PUBLIC.SHARD (shard_id, shard_hash) VALUES
-(1, X'8dd2cb2fcd453c53b3fe53790ac1c104a6a31583e75972ff62bced9047a15176'),
-(2, X'a3015d38155ea3fd95fe8952f579791e4ce7f5e1e21b4ca4e0c490553d94fb7d'),
-(3, X'931A8011F4BA1CDC0BCAE807032FE18B1E4F0B634F8DA6016E421D06C7E13693')
+INSERT into PUBLIC.SHARD (shard_id, shard_hash, shard_height) VALUES
+(1, X'8dd2cb2fcd453c53b3fe53790ac1c104a6a31583e75972ff62bced9047a15176', 1),
```

Comment:
1, 2 , 3 - not so good height values for shards. Make it more real.

Reply:
It's a fake shards, we don't need more real values here. It works quite good now.